src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static boolean diff_string_check(String[] arr, String toCheckValue)
+    {
+        boolean test = true;
+
+        for (int i = 0; i < arr.length; i++)
+        {
+            if (toCheckValue.equals(arr[i]))
+            {
+                test = false;
+                break;
+            }
+            else if (arr[i] == null)
+            {
+                break;
+            }
+        }
```

Comment:
Perhaps this violates the coding standards. You can write the code as follows:
if(){
---
--
} else if(){
---
---
} else{
--
--
}